readme.txt

```diff
@@ -64,6 +64,18 @@ OneSignal is trusted by over 1.8M+ developers and marketing strategists. We powe
 
 == Changelog ==
 
+= 3.0.0 =
+WARNING: this update contains breaking changes.
+We are moving all OneSignal plugin settings to the OneSignal.com dashboard.
+If you have made any OneSignal custom-code modifications (not via UI) updating
```

Comment:
If you have made any code modifications to your Wordpress installation to directly interact with the OneSignal SDK, this version will break your implementation. Please revert your code modifications prior to upgrading and use our improved dashboard functionality to configure your OneSignal integration instead.